cli/command/formatter/stats.go

```diff
@@ -37,11 +35,80 @@ type ContainerStatsAttrs struct {
 	PidsCurrent      uint64 // Not used on Windows
 }
 
-// ContainerStats represents the containers statistics data.
+// ClearContainerStats represents the statistics data collected from a container.
+type ClearContainerStats struct {
```

Comment:
@thaJeztah  `ContainerStats` is used for collecting the container statistics while `ClearContainerStats` (now `ContainerStatsRecord) is used to display the statistics on the screen.

Because of that `ContainerStats` has both mutex and an error field.